app/helpers/sprangular/application_helper.rb

```diff
@@ -22,20 +22,38 @@ def js_environment
         end
       ]
 
-      {env: Rails.env,
+      {
+        env: Rails.env,
         config: {
           site_name: store.seo_title || store.name,
           logo:      asset_path(config.logo),
+          locale:    I18n.locale,
+          currency:    Money::Currency.table[current_currency.downcase.to_sym],
+          supported_locales: supported_locales,
           default_country_id: config.default_country_id,
           facebook_app_id: ENV['FACEBOOK_APP_ID'],
           payment_methods: payment_methods,
           image_sizes: Spree::Image.attachment_definitions[:attachment][:styles].keys,
           product_page_size: Spree::Config.products_per_page
         },
+        translations: current_translations[:sprangular],
         templates: templates
       }
     end
 
+    def supported_locales
+      if Object.const_defined?('SpreeI18n')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.